tests/osaka/eip7692_eof_v1/eip4750_functions/test_callf_execution.py

```diff
@@ -252,3 +257,102 @@ def test_callf_with_inputs_stack_overflow(
         ),
         container_post=Account(storage={slot_code_worked: 0}),
     )
+
+
+@pytest.mark.parametrize(
+    ("stack_height", "inputs", "outputs", "failure"),
+    (
+        pytest.param(1020, 1, 3, False, id="1020"),
+        pytest.param(1021, 1, 3, True, id="1021"),
```

Comment:
```suggestion
        pytest.param(1020, 1, 3, False, id="success"),
        pytest.param(1021, 1, 3, True, id="failure"),
```
if we just have these two now (do I understand correctly, the intention is that `1022` would not validate anymore? Or are there other combinations of params which would validate and make sense to test?

Reply:
The essential test is pass below, and fail above.  I was considering messing with the input/output levels, but determined it was not essential to the test.  I took input and output of the params, and just have stack size and wether we expect failure.  Seemed to communicate intent better than embedding what a failure is into the text code.